apps/graphql/src/apps/itinerary/dataloaders/Itinerary.js

```diff
@@ -54,10 +55,21 @@ const sanitizeItinerary = (response: ItineraryApiResponse): Itinerary => {
    * @TODO - `price.currency` - Tequila API currently always returns total price
    * in EUR, should be unified with search results where currency can be variable
    */
+  const currency = response.currency;
   const sectors = sanitizeSectors(response.flights);
   const type = getItineraryType(sectors);
   const departure = getItineraryDeparture(sectors);
   const arrival = getItineraryArrival(type, sectors);
+  const holdBagOptions = getHoldBagOptions(
+    response.bags_price,
+    currency,
+    response.luggage?.[4],
```

Comment:
The Tequila API giving back an array of numbers and not an object with keys is 😞 ... maybe we can have a type `LuggageOptions` that is:
```
hand_length
hand_width
hand_height
hand_weight
hold_weight
hold_length
hold_width
hold_height
note = note of hold luggage
hand2_length
hand2_width
hand2_height
hand2_weight
hand2_note
hand_note
```
And have a method to map the array of luggage params to this object so we at least have it somewhat documented as fields not as positions (?¿)

Reply:
I wanted to avoid fixing Tequila problems (because it will probably not end with just this 🙂). But you are right. Extra type and parser function in this case will not hurt and it will be more readable.
I'll update it.

Reply:
Refactored and added "sanitizer" which makes it little bit more readable.